lib/rsocket/requester.rb

```diff
@@ -57,6 +57,15 @@ def dispose
       close_connection(true)
     end
 
+    # @param error_frame [RSocket:ErrorFrame]
+    def error_handler(error_frame)
```

Comment:
nitpick: verb names for methods e.g. receive_response not response_receiver. In the same vein handle_error not error_handler... but we can fix in future